### PR TITLE
calculate size for each mimetype

### PR DESCRIPTION
**Pull request by emrekuecuek**
implementation of #2

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <licence>AGPL</licence>
     <author>Emre Küçük</author>
     <dependencies>
-        <owncloud min-version="9.0" max-version="10"/>
+        <owncloud min-version="9.0" max-version="10.0.0"/>
```
Comment (karakayasemi) [inline appinfo/info.xml]: max version should be "10"

File: lib/AppInfo/Application.php [hunk 2]
```diff
@@ -3,11 +3,12 @@
 use \OCP\AppFramework\App;
 use OCP\IContainer;
 
-class Application extends App {
+class Application extends App{
```
Comment (karakayasemi) [inline lib/AppInfo/Application.php]: " space after app"

File: lib/Controller/PageController.php [hunk 3]
```diff
@@ -6,14 +6,39 @@
  * Time: 21:06
  */
 namespace OCA\Analysis_app\Controller;
+use OCA\Analysis_app\FileModel;
 use OCP\AppFramework\Controller;
 use OCP\AppFramework\Http\TemplateResponse;
+use OCP\IRequest;
+class PageController extends Controller
+{
```
Comment (karakayasemi) [inline lib/Controller/PageController.php]: it should "Controller {"

File: lib/Controller/PageController.php [hunk 4]
```diff
@@ -6,14 +6,39 @@
...
+    public function __construct(
+        $appName,
+        IRequest $request,
+        FileModel $fileModel
 
+    ){
+        parent::__construct($appName, $request);
+        $this->fileModel = $fileModel;
+    }
     public function index() {
```
Comment (karakayasemi) [inline lib/Controller/PageController.php]: add phpdoc

**Review by karakayasemi — CHANGES_REQUESTED** [review]

**Comment by karakayasemi** [comment]
?

File: lib/Controller/PageController.php (hunk 4 above)
Comment (emrekuecuek) [reply lib/Controller/PageController.php]: for index function?

**Comment by karakayasemi** [comment]
now squash your commits and give proper commit message (IMHO first message was fine)

**Review by karakayasemi — APPROVED** [review]

**Pull request merged**